### PR TITLE
Change font color of Duration and Time notes

### PR DESCRIPTION
**Pull request by MuhammadAzka9**
in this case i changed the font color of the Duration and Time notes 
URL https://raw.githack.com/MuhammadAzka9/open-source/features/add-ChangeFontColor/en_ID/#!pages/internship.md#Step_0_-_Introductory_steps

### CODE REVIEW

**Comment by empeje** [comment]
You have the same issue with @HananAskarim, see https://github.com/kulkultech/open-source/pull/82#issuecomment-826259336

**Comment by empeje** [comment]
I have a comment here as well (https://github.com/kulkultech/open-source/issues/65#issuecomment-826459648)

File: en_ID/pages/internship.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ We are a tech-based company and software engineering is at the heart of our comp
 
 ## Duration and time commitment
 
-NOTE: This is an intensive internship, a dedicated amount of effort, and time are required.
+<font color="red">NOTE: This is an intensive internship, a dedicated amount of effort, and time are required.
```
Comment (empeje) [inline en_ID/pages/internship.md]: You can use this instead:

```markdown
NOTE:
This is an intensive internship, a dedicated amount of effort, and time are required.
```

**Comment by MuhammadAzka9** [comment]
I change the font color of note into attention, please review ya
font

File: en_ID/index.html [hunk 2]
```diff
@@ -83,6 +83,7 @@
         /* md-text is not of md-col-* but needs the spacing */
         margin-left: 15px;
         margin-right: 15px;
+        color:#8b0000;
```
Comment (empeje) [inline en_ID/index.html]: What is this for?

Comment (MuhammadAzka9) [reply en_ID/index.html]: oh sorry, i forget to delete this, because i just wanna try it

Comment (empeje) [reply en_ID/index.html]: Could you please update?

Comment (MuhammadAzka9) [reply en_ID/index.html]: i have done to update, please review

**Review by empeje — APPROVED** [review]

**Comment by empeje** [comment]
Nicely done 🎉

**Pull request merged**